{"identifier":{"url":"doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/failed","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/app.soulver.core\/documentation\/SoulverCore","doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult"]]},"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"failed","kind":"identifier"}],"symbolKind":"case","title":"EvaluationResult.failed","externalID":"s:11SoulverCore16EvaluationResultO6failedyA2CmF","modules":[{"name":"SoulverCore"}],"roleHeading":"Case","role":"symbol"},"seeAlsoSections":[{"identifiers":["doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/none","doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/pending","doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/error(_:)"],"generated":true,"title":"Nil answer cases"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/soulvercore\/evaluationresult\/failed"]}],"kind":"symbol","sections":[],"abstract":[{"text":"The network could not deliver a result (perhaps due to connectivity problems or a service outage)","type":"text"}],"schemaVersion":{"major":0,"patch":0,"minor":3},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"failed"}]}]}],"references":{"doc://app.soulver.core/documentation/SoulverCore/EvaluationResult/none":{"url":"\/documentation\/soulvercore\/evaluationresult\/none","kind":"symbol","identifier":"doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/none","title":"EvaluationResult.none","abstract":[{"text":"A null result","type":"text"}],"type":"topic","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"none"}]},"doc://app.soulver.core/documentation/SoulverCore/EvaluationResult/failed":{"url":"\/documentation\/soulvercore\/evaluationresult\/failed","kind":"symbol","identifier":"doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/failed","title":"EvaluationResult.failed","abstract":[{"text":"The network could not deliver a result (perhaps due to connectivity problems or a service outage)","type":"text"}],"type":"topic","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"failed"}]},"doc://app.soulver.core/documentation/SoulverCore/EvaluationResult":{"identifier":"doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"EvaluationResult","kind":"identifier"}],"url":"\/documentation\/soulvercore\/evaluationresult","kind":"symbol","type":"topic","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"EvaluationResult"}],"title":"EvaluationResult","abstract":[{"text":"Represents the underlying data of a result of a SoulverCore calculation","type":"text"}]},"doc://app.soulver.core/documentation/SoulverCore":{"kind":"symbol","url":"\/documentation\/soulvercore","identifier":"doc:\/\/app.soulver.core\/documentation\/SoulverCore","type":"topic","title":"SoulverCore","abstract":[{"text":"A powerful natural language calculation engine","type":"text"}],"role":"collection"},"doc://app.soulver.core/documentation/SoulverCore/EvaluationResult/pending":{"url":"\/documentation\/soulvercore\/evaluationresult\/pending","kind":"symbol","identifier":"doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/pending","title":"EvaluationResult.pending","abstract":[{"text":"A result that must be fetched asyncronously (probably over a network)","type":"text"}],"type":"topic","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"pending"}]},"doc://app.soulver.core/documentation/SoulverCore/EvaluationResult/error(_:)":{"url":"\/documentation\/soulvercore\/evaluationresult\/error(_:)","kind":"symbol","identifier":"doc:\/\/app.soulver.core\/documentation\/SoulverCore\/EvaluationResult\/error(_:)","title":"EvaluationResult.error(_:)","abstract":[],"type":"topic","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"error","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"EvaluationError","preciseIdentifier":"s:11SoulverCore15EvaluationErrorO"},{"kind":"text","text":")"}]}}}